src/content/docs/release-notes/agent-release-notes/ruby-release-notes/ruby-agent-8140.mdx

```diff
@@ -3,6 +3,9 @@ subject: Ruby agent
 releaseDate: '2022-12-14'
 version: 8.14.0
 downloadLink: 'https://rubygems.org/downloads/newrelic_rpm-8.14.0.gem'
+features: ["Speed up GUID generation", "Add support for Rails ActiveSupport::Notifications for custom events"]
+bugs: ["Support older versions of the RedisClient gem, handle unknown Redis database index", "Defer agent startup in Rails until after application-defined initializers have run"]
```

Comment:
Specific versions may be helpful for the RedisClient entry

Reply:
[Added more info](https://github.com/newrelic/docs-website/pull/11703/commits/f755804b3a1fea8abcd87d362cc92a4c34c3ea03#diff-c5491b6773b1708e6a4afc3247b1ec5d2e3383556ad774a70ee07523af221d58R7)